pyam/core.py

```diff
@@ -477,6 +484,8 @@ def append(self, other, ignore_meta_conflict=False, inplace=False, **kwargs):
             any meta columns present in `self` and `other` are not identical.
         inplace : bool, default False
             If True, do operation inplace and return None
+        verify_integrity : bool, default True
```

Comment:
Per numpydoc convention, use optional instead of the value (which is in the signature anyway, see section 4 at https://numpydoc.readthedocs.io/en/latest/format.html

Can you update this for all kwargs `append()` function? (The docstrings are currently not consistent, so I try to harmonize it towards numpydoc whenever a function is touched).